guacamole/src/main/frontend/src/app/auth/service/authenticationService.js

```diff
@@ -115,9 +111,10 @@ angular.module('auth').factory('authenticationService', ['$injector',
             // Always store in cache
             cachedResult = data;
 
-            // Persist result past tab/window closure ONLY if not anonymous
+            // Persist auth token past tab/window closure ONLY if not anonymous
             if (data.username !== AuthenticationResult.ANONYMOUS_USERNAME)
-                localStorageService.setItem(AUTH_STORAGE_KEY, data);
+                localStorageService.setItem(
+                        AUTH_TOKEN_STORAGE_KEY, data.authToken);
```

Comment:
I think we should document that:

* Only the auth token is persisted locally, while the rest of the auth result is cached only in memory.
* The rest of the authentication result can be obtained by reauthenticating with the persisted token, which can be retrieved with `getCurrentToken()`.

Reply:
Done